keyboards/dnworks/9973/info.json

```diff
@@ -0,0 +1,121 @@
+{
+    "keyboard_name": "DN 997.3",
+    "maintainer": "itsme-zeix",
+    "manufacturer": "dnworks",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4C23",
+        "pid": "0x2936",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true    
+    },
+    "indicators": {
+        "caps_lock": "GP27",
+        "scroll_lock": "GP1",
+        "on_state": 1
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP2", "GP3", "GP4", "GP29", "GP12", "GP13"],
+        "cols": ["GP28", "GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP6", "GP5", "GP0"]
+    },
+    "layouts": {
```

Comment:
@itsme-zeix 
Regarding physical layout of this board, does production PCB (can only find images of prototype) have ISO layout support? i.e. could equivalent ISO layout be added

Reply:
Yes, it does support ISO! The ISO Enter is mapped to the ANSI Enter [3, 12]. ISO Non-US-Hash is mapped to the ANSI Pipe key [2, 13]. In that case, would a community layout need to be added for ISO, presumably adjusted for key sizing/positions?

On that note, how do you guys usually generate the matrix diagram and community layouts? Would that be by hand or is there some tool available?

Reply:
> Yes, it does support ISO! The ISO Enter is mapped to the ANSI Enter [3, 12]. ISO Non-US-Hash is mapped to the ANSI Pipe key [2, 13]. In that case, would a community layout need to be added for ISO, presumably adjusted for key sizing/positions?

edited related suggestion for ease 😉

> On that note, how do you guys usually generate the matrix diagram and community layouts?

Community Layouts and their templated information are available in directory `layouts/default/`:
- Key sequence/order, key spacings, and key widths for a given Community Layout are found in respective layout's `info.json` file
  - visual representation of a Community Layout can be copy-pasted from `layout.json` into "Raw Data" tab of [KLE](http://www.keyboard-layout-editor.com/#/) 
- ASCII diagram templates, for `matrix_diagram.md`, found in `layouts/default/readme.md`

> Would that be by hand or is there some tool available?

To my knowledge, and happy to be corrected, is done by hand - a GUI tool and/or QMK CLI command that generates multiple layout information to a .json file would be 🤩

Reply:
No worries, thank you for the changes made!

Noted on the community layouts/matrix diagrams - I will try to incorporate these in future PRs. Also, never knew about the configurator's preview function, this sounds really, really useful. Perhaps I may come up with some tool as a side project to aid in the firmware generation after my finals (assuming enough motivation). Thanks for the review again!